volume/volume_test.go

```diff
@@ -0,0 +1,130 @@
+package volume
+
+import (
+	"runtime"
+	"strings"
+	"testing"
+)
+
+func TestParseMountSpec(t *testing.T) {
+	var (
+		valid   []string
+		invalid map[string]string
+	)
+
+	if runtime.GOOS == "windows" {
+		valid = []string{
+			`d:\`,
+			`d:`,
+			`d:\path`,
+			`d:\path with space`,
+			`d:\pathandmode:rw`,
+			// TODO Windows post TP4 - readonly support `d:\pathandmode:ro`,
+			`c:\:d:\`,
+			`c:\windows\:d:`,
```

Comment:
No big issue, but should there be a mixed-case path in the list? e.g. Uppercase `C:\Windows\`. Also not sure if these are important but a `C:\Program Files (x86)\` could be good so we don't regress at some point (I hate those paths, but they're quite common, lol)